src/librbd/plugin/WriteLogImageCache.cc

```diff
@@ -42,7 +42,7 @@ void WriteLogImageCache<I>::init(I* image_ctx, Api<I>& api,
                                  PluginHookPoints& hook_points_list,
                                  Context* on_finish) {
   bool rwl_enabled = image_ctx->config.template get_val<bool>(
-    "rbd_rwl_enabled");
+    "rbd_persistent_cache_enabled");
```

Comment:
Do we have such option? I thought it was changed to `rbd_persistent_cache_mode`.

Reply:
I misunderstood our convo. I thought you needed it. As of today `rbd_persistent_cache_enabled` doesn't exist in `src/common/options.cc`. Can remove this option and this code can be replaced with `is_pwl_enabled` util. Will fix this.